prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0001.fhir

```diff
@@ -6508,32 +6418,23 @@
         "text" : "Gentamicin.high potency [Susceptibility] by Minimum inhibitory concentration (MIC)"
       },
       "subject" : {
-        "reference" : "Patient/7806900e-8e0c-4e1d-bd7a-648fa966d935"
+        "reference" : "Patient/f39a0f20-ba1f-48f2-934f-35e2a3c51f10"
       },
       "encounter" : {
-        "reference" : "Encounter/011de763-d0e8-49fc-a662-e5cc348d5660"
+        "reference" : "Encounter/ebcf14ca-78e2-486c-860a-47df285253a2"
       },
       "effectiveDateTime" : "2028-08-07T16:36:01-06:00",
       "issued" : "2028-08-07T16:36:01-06:00",
       "performer" : [ {
-        "reference" : "PractitionerRole/e035e0d0-31be-4544-b823-648ecdd82703"
-      } ],
-      "valueString" : "SYN-S",
-      "interpretation" : [ {
-        "coding" : [ {
-          "system" : "http://terminology.hl7.org/CodeSystem/v2-0078",
-          "code" : "S",
-          "display" : "Susceptible"
```

Comment:
This got removed.  Same field as the previous one I think.

Reply:
I'm back!